Automattic/jetpack
# General: update module headings

#### Changes proposed in this Pull Request:

Update module headings to include all latest changes to module titles and descriptions.

#### Testing instructions:

* Not much to test here. Run `yarn build` while on this branch, no more changes should appear.

#### Proposed changelog entry for your changes:

* None
